PWGCF/FEMTOSCOPY/FemtoDream/AliFemtoDreamCorrHists.cxx

```diff
@@ -680,6 +690,22 @@ AliFemtoDreamCorrHists::AliFemtoDreamCorrHists(AliFemtoDreamCollConfig *conf,
                                               MixedmTEventName.Data(), *itNBins,
                                               *itKMin, *itKMax, 225, 0, 7.5);
         fPairs[Counter]->Add(fMixedEventmTDist[Counter]);
+
+        TString SamemTvsMultEventName = TString::Format("SEmTvsMultDist_Particle%d_Particle%d", iPar1,
+                                             iPar2);
+        fSameEventmTvsMultDist[Counter] = new TH2F(SamemTvsMultEventName.Data(),
+                                                   SamemTvsMultEventName.Data(), 
+                                                   225, 0, 7.5, multbins, 1, 
+                                                   multbins + 1);
+        fPairs[Counter]->Add(fSameEventmTvsMultDist[Counter]);
+
+        TString MixedmTvsMultEventName = TString::Format("MEmTvsMultDist_Particle%d_Particle%d", iPar1,
+                                              iPar2);
+        fMixedEventmTvsMultDist[Counter] = new TH2F(MixedmTvsMultEventName.Data(),
+                                                    MixedmTvsMultEventName.Data(), 
+                                                    225, 0, 7.5, multbins, 1, 
+                                                    multbins + 1);
+        fPairs[Counter]->Add(fMixedEventmTvsMultDist[Counter]);
```

Comment:
now you declare them whenever you do the regular old mT binning, but we really use this, in case we do the kStar vs. Multiplicity binning in mT bins. So this should be further up, when the condition fmTMultPlots == true.